src/options.ts

```diff
@@ -240,6 +244,11 @@ export function optionsFromParameter(parameter: string | undefined): Options {
     options.exportCommonSymbols = false;
   }
 
+  if (options.outputBeforeRequest || options.outputAfterResponse) {
+    options.outputTypeRegistry = true;
```

Comment:
Related to my question about doing `beforeRequest(request)`, and then letting the caller do a `$type` lookup if the need to, that would also mean I think we could remove this part, but `outputBeforeRequest` / `outputBeforeResponse` nudging `outputTypeRegistry` on.